LICENSE.txt

```diff
@@ -1,7 +1,7 @@
 
                                  Apache License
                            Version 2.0, January 2004
-                        http://www.apache.org/licenses/
+                        https://www.apache.org/licenses/
```

Comment:
In general, I think we're advised to keep the license text exactly as it is. And I notice that even the copy of the license that is served from https (that is, from https://www.apache.org/licenses/LICENSE-2.0) uses http: links.